.github/workflows/openvex.yml

```
@@ -0,0 +1,19 @@
+name: openvex
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
```

(nit) We use `v4` without pinning to a specific minor or patch version throughout the repository, so IMO it should be safe to do the same here, since we'd notice any bugs linked to it anyway.

```suggestion
        uses: actions/checkout@v4
```